roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+# defaults file for openshift_prometheus
+openshift_prometheus_state: present
+
+openshift_prometheus_namespace: prometheus
+
+openshift_prometheus_replicas: 1
+openshift_prometheus_node_selector: {"region":"infra"}
+
+# container resources
+openshift_prometheus_cpu_limit_prometheus: null
+openshift_prometheus_memory_limit_prometheus: null
+openshift_prometheus_cpu_requests_prometheus: null
+openshift_prometheus_memory_requests_prometheus: null
+openshift_prometheus_cpu_limit_alertmanager: null
+openshift_prometheus_memory_limit_alertmanager: null
+openshift_prometheus_cpu_requests_alertmanager: null
+openshift_prometheus_memory_requests_alertmanager: null
+openshift_prometheus_cpu_limit_alert_buffer: null
+openshift_prometheus_memory_limit_alert_buffer: null
+openshift_prometheus_cpu_requests_alert_buffer: null
+openshift_prometheus_memory_requests_alert_buffer: null
+openshift_prometheus_cpu_limit_oauth_proxy: null
+openshift_prometheus_memory_limit_oauth_proxy: null
+openshift_prometheus_cpu_requests_oauth_proxy: null
+openshift_prometheus_memory_requests_oauth_proxy: null
```

Comment:
@joelddiaz I wasn't sure if I should add limits and requests for each container and what should be the default values. Currently, I added null defaults for all. could you assist with the defaults?